application.py

```diff
@@ -4,9 +4,10 @@
 from app import create_app
 from flask.ext.script import Manager, Server
 
+port = int(os.environ.get('PORT', 5002))
 application = create_app(os.getenv('DM_ENVIRONMENT') or 'development')
 manager = Manager(application)
-manager.add_command("runserver", Server(port=5002))
+manager.add_command("runserver", Server(host='0.0.0.0', port=port))
```

Comment:
We can customize port with `-p` argument, I assume this is part of the accidental commit of heroku-related changes